code/game/machinery/vending/marine_vending.dm

```diff
@@ -111,6 +111,8 @@
 					// /obj/item/weapon/gun/shotgun/combat = 1,
 					/obj/item/explosive/mine = 2,
 					/obj/item/storage/box/nade_box = 2,
+					/obj/item/storage/box/nade_box/M15 = 1 ,
+ 					/obj/item/storage/box/nade_box/incendiary = 0,
```

Comment:
If the default value is to be zero and you want to still keep it useful for admin events, you should move it under the contraband list.